### DISCUSSION TITLE
can't upload local image

### DISCUSSION THREAD

**Discussion by cerebellumking** [community/community#50303] [GitHub Education] [answered]
why I can't upload the local image during my revivification
<img width="751" alt="image" src="https://user-images.githubusercontent.com/83447519/225859410-78c831ae-e3a8-496c-9407-e6b5da11c5e3.png">

**Comment by Anmol-Baranwal** [comment]
If you are having trouble uploading a local image to GitHub during your verification process, it's possible that you are encountering one of the following issues:

1. GitHub has a file size limit of 100 MB for files uploaded through the web interface. If your image is larger than 100 MB, you won't be able to upload it. In this case, you may need to resize the image or compress it to reduce its size.

2. GitHub supports a variety of file types, including PNG, JPEG, GIF, and SVG. Make sure that your image is in a supported format.

3. GitHub has some restrictions on file names. Make sure that your file name doesn't include any special characters or spaces. You may need to rename the file before uploading it.

4. If you have a slow or unstable internet connection, you may have difficulty uploading large files. Try uploading the file from a location with a better internet connection.

> I hope this helps you upload your image successfully!

**Reply by Anmol-Baranwal** [reply]
@cerebellumking 
I just wanted to check in and see if you had any further questions or concerns about my answer on Github. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by cerebellumking** [reply]
emmm, my problem is that I haven't the button of uploading image

**Reply by Anmol-Baranwal** [reply]
You should try to contact GitHub support [here](https://support.github.com/contact?tags=docs-generic)

**Reply by samus-aran** [reply]
@Anmol-Baranwal 👋 It's awesome seeing you help out! We do want to highlight that for these types of enquiries we do not direct users to support 👍

**Reply by Anmol-Baranwal** [reply]
Okay, I thought it would help :) 
I will keep a note of this. Thanks for letting me know

**Reply by cerebellumking** [reply]
thanks for your help

**Comment by samus-aran** [comment]
Hi @cerebellumking 👋 

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance

**Reply by cerebellumking** [reply]
thanks for your help

**Comment by cerebellumking** [comment, marked as answer]
My solution is that I print the verification report and use the camera of my Macbook to capture.☘️
